Bridge crashes when mime type of file cannot be determined.

When POSTing the example webhook payload the bridge sends users, the bridge will crash with the following error:

```
May-2-2020 19:13:46.543 +00:00 info [utils] No content-type given by server, guessing based on file name.
[ERROR] TypeError: mime.lookup is not a function
    at ClientRequest.ht.get (/src/utils.js:44:36)
    at Object.onceWrapper (events.js:286:20)
    at ClientRequest.emit (events.js:198:13)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:565:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:111:17)
    at Socket.socketOnData (_http_client.js:451:20)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
```

Based on my investigation, it seems https://github.com/turt2live/matrix-appservice-webhooks/blob/275cc31750e59260d02938058be5e590e1958ab2/src/utils.js#L44 is using syntax from the `mime` v1 package when `packages.json` has `mime` pinned to v2.

The documentation for `mime` states:
Version 2 is a breaking change from 1.x as the semver implies. Specifically:
lookup() renamed to getType()

--- a/src/utils.js
+++ b/src/utils.js
@@ -41,7 +41,7 @@ function uploadContentFromUrl(bridge, url, id, name) {
                 contenttype = res.headers["content-type"];
             } else {
                 LogService.info("utils", "No content-type given by server, guessing based on file name.");
-                contenttype = mime.lookup(url);
+                contenttype = mime.getType(url);
             }
 
             if (name == null) {

